src/components/search/HeaderSearch.jsx

```diff
@@ -88,6 +94,16 @@ const HeaderSearch = () => {
           onBlur={handleUriChange}
         >
           <option value={sinopiaSearchUri}>Sinopia</option>
+          <option value={`${sinopiaSearchUri}/Work`}>
+            Sinopia BIBFRAME work resources
+          </option>
+          <option value={`${sinopiaSearchUri}/Instance`}>
+            Sinopia BIBFRAME instance resources
+          </option>
+          <option value={`${sinopiaSearchUri}/Item`}>
+            Sinopia BIBFRAME Items
```

Comment:
Can you please change to `Sinopia BIBFRAME item resources`?

Reply:
Ticket #3263 specifies this as written. Do you want to amend the ticket?